src/platforms/kotlin-multiplatform/common/install/swift-package-manager.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Swift Package Manager (SPM)
+description: "Learn about installing the Sentry Kotlin Multiplatform SDK with Swift Package Manager."
+---
+
+<Note>
+  We recommend using version 8.4.0 of the Sentry Cocoa SDK, as we cannot
+  guarantee correctness with other versions.
+</Note>
+
+Add the Sentry Cocoa SDK as a package in Xcode in your iOS app via **File > Add Packages**.
+Enter the Git repo URL in the search field:
+
+```text
+https://github.com/getsentry/sentry-cocoa.git
```

Comment:
Can we pin point the 8.4.0 tag instead? otherwise there will be version mismatch

Reply:
Added some extras to make this clear